Sources/Models/Connections/Segue.swift

```diff
@@ -80,4 +84,89 @@ public struct Segue: IBDecodable, ConnectionProtocol {
         }
     }
 
+    public enum ModalPresentationStyle: XMLAttributeDecodable, KeyDecodable, Equatable {
+        case automatic, fullScreen, pageSheet, formSheet, currentContext
+        case custom, overFullScreen, overCurrentContext, blurOverFullScreen, popover, none
+
+        public func encode(to encoder: Encoder) throws { fatalError() }
+
+        static func decode(_ attribute: XMLAttribute) throws -> Segue.ModalPresentationStyle {
+            switch attribute.text {
+            case "automatic": return .automatic
+            case "fullScreen": return .fullScreen
+            case "pageSheet": return .pageSheet
+            case "formSheet": return .formSheet
+            case "currentContext": return .currentContext
+            case "custom": return .custom
+            case "overFullScreen": return .overFullScreen
+            case "overCurrentContext": return .overCurrentContext
+            case "blurOverFullScreen": return .blurOverFullScreen
+            case "popover": return .popover
+            case "none": return .none
+            default:
+                return .none
+            }
+        }
+
+        public static func == (left: ModalPresentationStyle, right: ModalPresentationStyle) -> Bool {
+            switch (left, right) {
+            case (.automatic, .automatic):
+                return true
+            case (.fullScreen, .fullScreen):
+                return true
+            case (.pageSheet, .pageSheet):
+                return true
+            case (.formSheet, .formSheet):
+                return true
+            case (.currentContext, .currentContext):
+                return true
+            case (.overFullScreen, .overFullScreen):
+                return true
+            case (.overCurrentContext, .overCurrentContext):
+                return true
+            case (.blurOverFullScreen, blurOverFullScreen):
+                return true
+            case (.popover, .popover):
+                return true
+            case (.custom, .custom):
+                return true
+            case (.none, .none):
+                return true
+            default:
+                return false
+            }
+        }
+    }
+
+    public enum ModalTransitionStyle: XMLAttributeDecodable, KeyDecodable, Equatable {
+        case coverVertical, flipHorizontal, crossDissolve, partialCurl
+
+        public func encode(to encoder: Encoder) throws { fatalError() }
+
+        static func decode(_ attribute: XMLAttribute) throws -> Segue.ModalTransitionStyle {
+            switch attribute.text {
+            case "coverVertical": return .coverVertical
+            case "flipHorizontal": return .flipHorizontal
+            case "crossDissolve": return .crossDissolve
+            case "partialCurl": return .partialCurl
+            default:
+                throw IBError.elementNotFound
+            }
+        }
+
+        public static func == (left: ModalTransitionStyle, right: ModalTransitionStyle) -> Bool {
```

Comment:
I think this `==` implementation is automatically synthesized by compiler, so it maybe not necessary

Reply:
Yes it’s true
There is no custom(string) here!